mlt 7.22.0 does not compile with libxml2 2.12.0

mlt 7.22.0 tarball does not compile with libxml2 2.12.0 as:

```
/builddir/build/BUILD/mlt-7.22.0/src/modules/jackrack/jack_rack.c: In function ‘jack_rack_open_file’:
/builddir/build/BUILD/mlt-7.22.0/src/modules/jackrack/jack_rack.c:343:9: error: implicit declaration of function ‘xmlParseFile’; did you mean ‘xmlSaveFile’? [-Werror=implicit-function-declaration]
```

libxml2 2.12.0 does some refactoring wrt function prototypes places, and this error is probably due to:
https://gitlab.gnome.org/GNOME/libxml2/-/commit/d1336fd39319b20c56e4dc5e7d5b1098a88eaff3

--- a/src/modules/jackrack/jack_rack.c
+++ b/src/modules/jackrack/jack_rack.c
@@ -29,6 +29,7 @@
 #include <ctype.h>
 
 #include <ladspa.h>
+#include <libxml/parser.h>
 #include <libxml/tree.h>
 
 #include "jack_rack.h"

